css/_uswds-theme-custom-styles.scss

```diff
@@ -2212,3 +2212,197 @@ h6 + div:not(.usa-accordion-content) {
 pre {
   white-space: pre-wrap;
 }
+
```

Comment:
This is largely the code from https://github.com/uswds/uswds/issues/896 except re-styled and using flex instead of absolute positioning